crates/revm/src/inspector.rs

```diff
@@ -47,70 +46,87 @@ pub trait Inspector<DB: Database> {
     /// # Example
     ///
     /// To get the current opcode, use `interp.current_opcode()`.
-    fn step(
-        &mut self,
-        _interp: &mut Interpreter,
-        _data: &mut EVMData<'_, DB>,
-    ) -> InstructionResult {
+    #[inline]
+    fn step(&mut self, interp: &mut Interpreter, data: &mut EVMData<'_, DB>) -> InstructionResult {
+        let _ = interp;
+        let _ = data;
         InstructionResult::Continue
     }
 
     /// Called when a log is emitted.
+    #[inline]
     fn log(
         &mut self,
-        _evm_data: &mut EVMData<'_, DB>,
-        _address: &B160,
-        _topics: &[B256],
-        _data: &Bytes,
+        evm_data: &mut EVMData<'_, DB>,
+        address: &B160,
+        topics: &[B256],
+        data: &Bytes,
     ) {
+        let _ = evm_data;
+        let _ = address;
+        let _ = topics;
+        let _ = data;
     }
 
     /// Called after `step` when the instruction has been executed.
     ///
     /// InstructionResulting anything other than [InstructionResult::Continue] alters the execution of the interpreter.
+    #[inline]
     fn step_end(
         &mut self,
-        _interp: &mut Interpreter,
-        _data: &mut EVMData<'_, DB>,
-        _eval: InstructionResult,
+        interp: &mut Interpreter,
+        data: &mut EVMData<'_, DB>,
+        eval: InstructionResult,
     ) -> InstructionResult {
+        let _ = interp;
+        let _ = data;
+        let _ = eval;
         InstructionResult::Continue
     }
 
     /// Called whenever a call to a contract is about to start.
     ///
     /// InstructionResulting anything other than [InstructionResult::Continue] overrides the result of the call.
+    #[inline]
     fn call(
         &mut self,
-        _data: &mut EVMData<'_, DB>,
-        _inputs: &mut CallInputs,
+        data: &mut EVMData<'_, DB>,
+        inputs: &mut CallInputs,
     ) -> (InstructionResult, Gas, Bytes) {
+        let _ = data;
+        let _ = inputs;
```

Comment:
hmm, what's the reason for that?

Reply:
Mainly trait docs and editor code actions, when you auto complete a trait function it copies the same formatting, names etc as the definition, at least on vscode

Reply:
Before
![image](https://github.com/bluealloy/revm/assets/57450786/0715b98b-3ae9-407e-b0fb-27d59845c79d)

After
![image](https://github.com/bluealloy/revm/assets/57450786/328d8484-b7c5-4091-811b-ad3bdf7f85b0)

Reply:
It's a very minor thing, idm if you want it the other way

Reply:
I see, TIL

I actually don't mind it then, defer to @rakita

Reply:
Looks strange, but the outcome is better, i like it